test/unit/commands/deploy/helper-test.js

```diff
@@ -15,6 +17,43 @@ describe('Commands deploy test - helper test', () => {
     const TEST_DO_DEBUG = false;
     const TEST_OPTIONS = { profile: TEST_PROFILE, doDebug: TEST_DO_DEBUG, ignoreHash: TEST_IGNORE_HASH };
 
+    describe('# test helper method - confirmProfile', () => {
+        afterEach(() => {
+            sinon.restore();
+        });
+
+        it('| profile does not exist, expect throw error', () => {
+            // setup
+            sinon.stub(ResourcesConfig.prototype, 'getProfile').returns(undefined);
+            try {
+                // call
+                helper.confirmProfile(TEST_PROFILE);
+            } catch (err) {
+                // verify
+                expect(err.message).equal(
```

Comment:
minor. recommended way to assert exceptions is with https://www.chaijs.com/api/bdd/#method_throw
for example:
`expect(() => helper.confirmProfile(TEST_PROFILE)).throw(CLiError, `CliError: ${errorMessage}`);`

Reply:
great to know, will apply this in this pr